x-pack/plugins/uptime/public/components/overview/alerts/monitor_expressions/translations.ts

```diff
@@ -56,6 +56,12 @@ export const alertFilterLabels = {
   }),
 };
 
+export const statusExpLabels = {
+  ENABLED_CHECKBOX: i18n.translate('xpack.uptime.alerts.monitorStatus.statusEnabledCheck.label', {
+    defaultMessage: 'Status check enabled',
```

Comment:
Can we change the copy here to just say 'Status check'. The check box makes it obvious that it's a boolean. CC @katrin-freihofner 

Same for the 'activity' section 

![image](https://user-images.githubusercontent.com/131427/86861828-9ae76880-c08d-11ea-888d-a7db9feaa5fa.png)

Reply:
Fixed in https://github.com/elastic/kibana/pull/70284/commits/42e8cd3d1e64a4bb018a815dcec9138780e9b81d.